### PR TITLE
SCMOD-6672/3: Fix database upgrade and add test

### PR DESCRIPTION
**Pull request by michael-bryson**
JIRA: https://portal.digitalsafe.net/browse/SCMOD-6673

Note that this is resurrecting the postgres database container and running an upgrade from version 2.5 to current as part of the projects build.
Dev build here: http://sou-jenkins2.hpeswlab.net/job/JobService/view/Developer/job/JobService~job-service~SCMOD-6673~CI/
The dev build is failing but that is expected due to https://portal.digitalsafe.net/browse/SCMOD-6672

### CODE REVIEW

File: job-service-postgres-container/pom.xml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+            <groupId>com.github.jobservice</groupId>
+            <artifactId>job-service-db</artifactId>
+            <version>${project.version}</version>
+            <classifier>jar-with-dependencies</classifier>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <!--Run integration tests.-->
```
Comment (dermot-hardy) [inline job-service-postgres-container/pom.xml]: Remove

File: job-service-postgres-container/pom.xml [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                        </goals>
+                        <configuration>
+                            <skip>false</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>start</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>build</goal>
```
Comment (dermot-hardy) [inline job-service-postgres-container/pom.xml]: Remove build goal

File: job-service-postgres-container/pom.xml [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+                                </log>
+                            </run>
+                        </image>
+                        
+                        <!-- Run jobservice postgres container against postgres database to upgrade jobservice schema. -->
+                        <image>
+                            <alias>job-service-postgres</alias>
+                            <name>${dockerJobServiceOrg}job-service-postgres${dockerProjectVersion}</name>
+                            <build>
+                                <from>cafapi/java-postgres:1.0</from>
```
Comment (dermot-hardy) [inline job-service-postgres-container/pom.xml:149]: 1-part

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: job-service-postgres-container/pom.xml (hunk 3 above)
Comment (michael-bryson) [reply job-service-postgres-container/pom.xml:149]: 1 part version not available: https://hub.docker.com/r/cafapi/java-postgres/tags

**Comment by michael-bryson** [comment]
Comments addressed.

**Review by dermot-hardy — APPROVED** [review]

**Comment by rahul-kulkarni73** [comment]
@dermot-hardy  You have already reviewed other changes from Michael. This change https://github.com/JobService/job-service/pull/92/commits/8bd1d4da697b577ef8600cbe8d2a90d447e2e693 will fix SCMOD-6672. 
That's the dev build - http://sou-jenkins2.hpeswlab.net/job/JobService/view/Developer/job/JobService~job-service~SCMOD-6673~CI/5/.

**Pull request merged**